app/scripts/setup.sh

```diff
@@ -74,18 +74,24 @@ function createSymlinks() {
     esac
 }
 
-function checkoutSilme() {
-    # Check out SILME library to a specific version (0.8.0)
-    if [ ! -d $libraries/silme/.hg ]
+function setupExternalLibraries() {
+    # Check out or update compare-locales library
+    version="RELEASE_1_2_1"
```

Comment:
silme was never updated and we needed a specific version, but maybe we will want to update compare-locales without having to do a new release? Of course we don’t want to automatically use the latest without testing it before, but maybe we should move the version number into the config file? We could then use it in both setup.sh and tmx_product.py